pnpm-workspace.yaml

```diff
@@ -10,3 +10,5 @@ packages:
   - 'src/satellite'
   # autodeployment
   - 'tools/*'
+
+link-workspace-packages: false
```

Comment:
This change effects both `satellite` and `eslint-config-telescope`. We made many ESLint changes recently for in eslint-config-telescope and the eslint-config-telescope in the npm registry hasn't picked up the new changes yet since we haven't done a release

Reply:
Actually, I think it makes sense to link eslint because we want eslint config to be reflected right away in subpackages. I link eslint in the new commit